### DISCUSSION TITLE
Commercial inquiry

### DISCUSSION THREAD

**Discussion by eiths** [tabler/tabler#1307] [General]
Hello, developer. Our company thinks that the tabler is very beautiful. Is it completely free if we use it commercially (no cost in the future)?

**Comment by codecalm** [comment]
hi @eiths, Tabler is on MIT license, so it's free for use in commercial projects :)

If you want to help us create and develop Tabler, you can become our sponsor and have a greater impact on the further development of the project :) https://github.com/sponsors/codecalm
